kb-footer/package.json

```diff
@@ -11,36 +14,22 @@
   },
   "dependencies": {
     "@lit/reactive-element": "^1.6.1",
-    "@test_kbdk/kb-fonts": "^1.2.0",
-    "@nordhealth/icons": "^2.0.0",
-    "@nordhealth/fonts": "^2.0.15",
-    "@nordhealth/css": "^2.4.1",
-    "@nordhealth/components": "^2.16.0",
-    "bootstrap": "^5.2.3",
     "lit": "^2.6.1",
     "lit-element": "^3.2.2",
-    "lit-html": "^2.6.1",
-    "@kb-dk/kb-icon": "^1.3.0",
-    "@kb-dk/kb-style": "^1.4.2"
+    "lit-html": "^2.6.1"
   },
-  "devDependencies": {
-    "copy-webpack-plugin": "^11.0.0",
-    "css-loader": "^6.7.3",
-    "html-loader": "^4.2.0",
-    "html-webpack-plugin": "^5.5.0",
-    "mini-css-extract-plugin": "^2.7.2",
-    "postcss": "^8.4.21",
-    "postcss-loader": "^7.0.2",
-    "postcss-preset-env": "^8.0.1",
-    "sass": "^1.58.2",
-    "sass-loader": "^13.2.0",
-    "style-loader": "^3.3.1",
-    "terser-webpack-plugin": "^5.3.6",
-    "ts-loader": "^9.4.2",
-    "typescript": "^4.9.5",
-    "webpack": "^5.75.0",
-    "webpack-bundle-analyzer": "^4.8.0",
-    "webpack-cli": "^5.0.1",
-    "webpack-dev-server": "^4.11.1"
-  }
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/kb-dk/kb-web-components.git"
+  },
+  "keywords": [
+    "kb",
+    "footer"
+  ],
+  "author": "Royal Danish library",
+  "license": "ISC",
```

Comment:
Correct license?

Reply:
I don't know much about licenses, but this was suggested but 'npm init' and I didn't change it.
What do you suggest?

Reply:
OK. I thnk we have some documentation about KB's license policy (haven't found it yet...) - let's leave this for now until I dig up the policy

Reply:
We use 'Apache License 2.0' for our open source software here on github